test/README.md

```diff
@@ -1,10 +1,11 @@
 # Test
 
-This directory contains tests and testing docs.
+This directory contains tests and testing docs for `Knative Eventing`.
 
-* [Unit tests](#running-unit-tests) currently reside in the codebase alongside the code they test
-* [End-to-end tests](#running-end-to-end-tests)
+* [Unit tests](#running-unit-tests) reside in the codebase alongside the code they test
+* [End-to-end tests](#running-end-to-end-tests) reside in [`/test/e2e`](./e2e)
 
+If you want to add more tests, see [adding_tests.md](./adding_tests.md).
```

Comment:
Isn't this on `e2e/README.md`?

Reply:
No. But i'm removing this line because it references the adding_tests.md file which doesn't exist. See our earlier discussion on this topic: https://github.com/knative/eventing/pull/416#discussion_r217448958